### DISCUSSION TITLE
Distribution of compiled executables

### DISCUSSION THREAD

**Discussion by oaustegard** [modular/modular#101] [General]
One of the things that stood out to me in the announcement was how this addresses distribution of Python applications, which today is tedious and/or inefficient. 

Where is this on the roadmap; how soon should we be able to use Mojo as a distribution vehicle for compiled Python/Mojo code?

**Comment by lattner** [comment]
We're working on getting mojo compiler out of the notebook, please stay tuned (we actually just mentioned this in a blog post)
